core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockSamDAO.scala

```diff
@@ -177,7 +177,9 @@ class MockSamDAO(dataSource: SlickDataSource)(implicit executionContext: Executi
     """{"client_email": "pet-110347448408766049948@broad-dsde-dev.iam.gserviceaccount.com", "client_id": "104493171545941951815"}"""
   )
 
-  override def getUserArbitraryPetServiceAccountKey(userEmail: String): Future[String] = ???
+  override def getUserArbitraryPetServiceAccountKey(userEmail: String): Future[String] = Future.successful(
+    """{"client_email": "pet-110347448408766049948@broad-dsde-dev.iam.gserviceaccount.com", "client_id": "104493171545941951815"}"""
```

Comment:
Is this real?  Should it be?

Reply:
I have no idea, but I can take a closer look and find out! I copied it over from the other mock methods 😅